<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeGeneratorGUIClass</class>
 <widget class="QMainWindow" name="CodeGeneratorGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code Generator for C++/SYCL Code</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>10</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Network File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>650</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cores:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberOfCores">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>650</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>760</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>270</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Native C Files(optional) - Header will be created for all given files, but imports cannot be checked!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sourceDirSelect">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>180</y>
      <width>821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="networkFileSelect">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Target_CPU_Cores">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>578</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Target CPU with Core limitation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="networkFileInput">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>30</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="No_SYCL">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>530</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No SYCL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nativeFileSelect_4">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>440</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Directory</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Target_GPU">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>610</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Target GPU</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>80</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="nativeFileSelect_3">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>400</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>700</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nativeFileSelect_2">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>360</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nativeFileInput_4">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>440</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FIFO_SIZE">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>650</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>180</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputDirInput">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>220</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nativeFileSelect_1">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>320</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nativeFileInput_3">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>400</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>270</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="SourceDirInput">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>130</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="outputDirSelect">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>220</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>530</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nativeFileInput_1">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>320</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>500</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>650</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>FIFO Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nativeFileInput_2">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>360</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="Target_CPU_no_limit">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>550</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Target CPU without Core limitation</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CodeGeneratorGUI.qrc"/>
 </resources>
 <connections/>
</ui>
